src/App/App.js

```diff
@@ -79,15 +79,18 @@ class App extends Component {
           showHomePage={this.showHomePage}
         />
         {page === 'Login' && 
-          <Login login={this.login} />}
+          <Login login={this.login} error={this.state.error}/>}
         {page === 'Home' && 
           <Main 
             movies={this.state.movies} 
             showMoviePage={this.showMoviePage} 
+            error={this.state.error}
           />}
         {page === 'MoviePage' && 
-          <MoviePage movie={this.state.singleMovie}/>}
-        {this.state.error && <Error error={this.state.error} />}
+          <MoviePage 
+          pageView={this.state.pageView}
+          movie={this.state.singleMovie}
+          error={this.state.error}/>}
```

Comment:
This looks much cleaner! At this point I'm assuming the Pages each can create their own errors / curious to see how pageView gets used within a page.